src/Package.js

```diff
@@ -96,6 +96,9 @@ export default class Package {
     // check if semantic versions are compatible
     if (semver.satisfies(actualVersion, expectedVersion)) {
       return true;
+    // or the dependency is specified as a relative folder
+    } else if (/^file:\.\.\//.test(dependency.version)) {
```

Comment:
that link shows that local directories will be serialized to `file:` in the package.json. I suppose we could loosen whats there a bit to look for only`file:`. That should work well.